Original URL: https://gitweb.gentoo.org/repo/gentoo.git/tree/media-libs/libquvi/files/libquvi-0.9.4-autoconf-2.70.patch

This patch will fix an FTBFS problem caused by update of GNU autoconf.
--- libquvi-0.9.4/configure.ac
+++ libquvi-0.9.4/configure.ac
@@ -8,7 +8,7 @@
 AC_INIT([libquvi], m4_esyscmd([./gen-ver.sh -c | tr -d '\n']),
         [http://quvi.sf.net/bugs/],[],[http://quvi.sf.net/])
 
-AC_DEFINE_UNQUOTED([BUILD_OPTS], "$@",
+AC_DEFINE_UNQUOTED([BUILD_OPTS], "$*",
   [Define to configure invocation command line options])
 
 # Interface
